### ISSUE TITLE
Optic pit

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9610]
**Preferred term label:**

**Synonyms**
optic nerve pit, or optic disc pit 

**Definition (free text, please give PubMed ID)**
Optic pits are congenital excavations of the optic nerve head 
https://pubmed.ncbi.nlm.nih.gov/27847606/
**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 616428 

PMID: 25910211  1/15

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
